rustls/src/lib.rs

```diff
@@ -470,7 +470,14 @@ use crate::crypto::ring as test_provider;
 // The public interface is:
 pub use crate::builder::{ConfigBuilder, ConfigSide, WantsVerifier, WantsVersions};
 pub use crate::common_state::{CommonState, IoState, Side};
-pub use crate::conn::{Connection, ConnectionCommon, Reader, SideData, Writer};
+pub use crate::conn::unbuffered::{
```

Comment:
Not sure this change makes sense in this commit?

Reply:
Okay, at least they're in a separate commit now, but I'm still confused. We already expose all of this stuff in the `unbuffered` module, around line 454?